docs/CHANGELOG/CHANGELOG-1.11.md

```diff
@@ -0,0 +1,147 @@
+- `karmada-controller-manager`: Fixed the issue that status aggregation against the resource template might be missed due to slow cache sync. ([#5318](https://github.com/karmada-io/karmada/pull/5318), @chaosi-zju)
+- `karmada-controller-manager`: fix the error of cluster status old condition update will overwrite the newest condition. ([#5227](https://github.com/karmada-io/karmada/pull/5227), @XiShanYongYe-Chang)
+- `karmada-controller-manager`: work status sync when work dispatching is suspended. ([#5403](https://github.com/karmada-io/karmada/pull/5403), @a7i)
+- `karmada-aggregated-apiserver`: User can append a "/" at the end when configuring the cluster's apiEndpoint. ([#5432](https://github.com/karmada-io/karmada/pull/5432), @spiritNO1)
+
+### Security
+- `Security Enhancement`: Introduced TLS certificate authentication mechanism to secure gRPC connections to `karmada-scheduler-estimator`. ([#5040](https://github.com/karmada-io/karmada/pull/5040), @zhzhuang-zju)
+    * The flags added to `karmada-scheduler-estimator` are:
+      --grpc-auth-cert-file
+      --grpc-auth-key-file 
+      --grpc-client-ca-file 
+      --insecure-skip-grpc-client-verify 
+    * The flags added to `karmada-scheduler` are:
+      --scheduler-estimator-ca-file
+      --scheduler-estimator-key-file
+      --scheduler-estimator-cert-file
+      --insecure-skip-estimator-verify
+    * The flags added to `karmada-descheduler` are:
+      --scheduler-estimator-ca-file
+      --scheduler-estimator-key-file
+      --scheduler-estimator-cert-file
+      --insecure-skip-estimator-verify
+    The added filed don't require any adoption during the process of upgrading from a previous version of Karmada, but gives an optional and recommended wait to secure the gRPC connection.
+- In this release, Karmada has invested significant effort in enhancing Karmada maturity based on [Clomonitor check sets](https://clomonitor.io/docs/topics/checks/). So far, we have achieved a score of [99](https://clomonitor.io/projects/cncf/karmada), and the last check (Signed releases) will be passed after 5 releases. This indicates that the Karmada community has made great strides in both security and maturity. (contributors: @zhzhuang-zju, @B1F030, @adiya7302, @Akash-Singh04, @RainbowMango)
+- Introduced [SBOM](https://www.aquasec.com/cloud-native-academy/supply-chain-security/sbom/) to release assests to enhance transparency around Karmada components and dependencies, bolster the security posture of our project. The composition and usage of SBOM can refer to [SBOM DOC](https://karmada.io/docs/next/administrator/security/verify-artifacts#sbom). ([#5110](https://github.com/karmada-io/karmada/pull/5110), @zhzhuang-zju)
+- Introduced [SLSA provenance file](https://slsa.dev/spec/v0.1/) to release assets, with which Karmada users verify the artifacts to prevent counterfeiting. ([#5178](https://github.com/karmada-io/karmada/pull/5178), @zhzhuang-zju)
+
+### Features & Enhancements
```

Comment:
Please group the notes by components. It makes it easier for people to find changes they are interested in.